### DISCUSSION TITLE
Initial release

### DISCUSSION THREAD

**Discussion by amitds1997** [amitds1997/remote-nvim.nvim#30] [Announcements]
First release of the plugin 🎉

## What's Changed
* feat(remote-nvim): Make SSH jobs async by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/1
* Add logic to find a free port by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/2
* Add LICENSE by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/3
* feat(remote-nvim): Add remote host configuration tracking by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/4
* Setup nvim server start by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/5
* fix(remote-setup): Fix port forwarding starting before setup by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/6
* feat(remote-login): Save password for session after first input by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/7
* feat(logging): Add logger by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/8
* feat(ssh-login): Add support for verifying new host by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/9
* docs(readme): Update README and TODO by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/10
* style: Add pre-commit, linters and formatters by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/11
* refactor(remote-nvim): Refactor existing code to abstract away providers by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/12
* feat: Add logging by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/13
* feat: Add logging by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/14
* feat: added progress reporter by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/15
* perf: Simplify checks by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/16
* feat: Improve user experience by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/17
* fix: launch of client while server is still is unavailable by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/18
* feat: add Neovim user commands by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/19
* Add README and docs by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/20
* chore: Improve UX further by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/21
* fix: broken docs CI by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/22
* fix: broken port forwarding handling by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/23
* fix: upload and download args by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/24
* fix: nasty little index issue leading to hanging SSH jobs by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/25
* ci: add release CI by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/26
* ci: fix doc and release generation CI by @amitds1997 in https://github.com/amitds1997/remote-nvim.nvim/pull/27
* chore(main): release 0.0.1 by @github-actions in https://github.com/amitds1997/remote-nvim.nvim/pull/28

## New Contributors
* @amitds1997 made their first contribution in https://github.com/amitds1997/remote-nvim.nvim/pull/1
* @github-actions made their first contribution in https://github.com/amitds1997/remote-nvim.nvim/pull/28

**Full Changelog**: https://github.com/amitds1997/remote-nvim.nvim/commits/v0.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/amitds1997/remote-nvim.nvim/releases/tag/v0.0.1'>Initial release</a>.</em>

**Comment by MohamedOsman1998** [comment]
I am not super familiar with neovim's --remote, which i think this runs under the hood correct?
but I wanted to ask, does nvim-dap and debugging with nvim-dap work well? I want to run on the remote but edit locally.

**Reply by amitds1997** [reply]
Yes, it should work. Anything that you could do locally, including using debugger should also work as long as your plugins are installed and working 👍 

https://github.com/amitds1997/remote-nvim.nvim/assets/29333147/bf43eeb4-80ed-4960-a289-1fa558051f86

**Reply by MohamedOsman1998** [reply]
great, ill definitely set that up then, great work and thanks for the video!
